File: 2020/Junior Problems/J3.py
ask = int(input(''))
list1 = []
for k in range(ask):
    ask2 = input('')
    cv = ask2.split(',')
    cv = [int(l) for l in cv]
    list1.append(cv)
list2 = [[],[],[],[]]
minx,miny,maxx,maxy = 1000,1000,0,0
for n in list1:
    k = n
    if k[1]<miny:
        miny=k[1]
        list2[1]=k
    if k[0]<minx:
        minx=k[0]
        list2[0]=k
    if k[0]>maxx:
        maxx=k[0]
        list2[2]=k
    if k[1]>maxy:
        maxy=k[1]
        list2[3]=k
num1 = [list2[0][0]-1,list2[1][1]-1]
num2 = [list2[2][0]+1,list2[3][1]+1]
print(str(num1[0])+','+str(num1[1]))
print(str(num2[0])+','+str(num2[1]))

File: 2020/C++/Junior Problems/J3.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int N;
    cin >> N;
    int min_x = 100;
    int min_y = 100;
    int max_x = 0;
    int max_y = 0;
    for (int i = 0; i < N; i++)
    {
        string line;
        cin >> line;
        int x = stoi(line.substr(0, line.find(",")));
        int y = stoi(line.substr(line.find(",")+1, -1));
        if (x < min_x)
        {
            min_x = x;
        }
        if (x > max_x)
        {
            max_x = x;
        }
        if (y < min_y)
        {
            min_y = y;
        }
        if (y > max_y)
        {
            max_y = y;
        }
    }
    min_x = (min_x > 0) ? min_x - 1 : min_x; // ternary operator in java
    min_y = (min_y > 0) ? min_y - 1 : min_y;
    max_x = (max_x < 100) ? max_x + 1 : max_x;
    max_y = (max_y < 100) ? max_y + 1 : max_y;
    cout << min_x << "," << min_y << endl;
    cout << max_x << "," << max_y << endl;
    return 0;
}
